### DISCUSSION TITLE
Add Progress bar and Speeds of HFS 2.3

### DISCUSSION THREAD

**Discussion by De-M-oN** [rejetto/hfs#136] [Ideas]
There is no way to see the progressbar and upload speed of someone uploading to me anymore?
Can you add this back?
And for the person who uploads it would be a nice to have if there would be a upload speed too.
Also the Logs section should automatically scroll the Log to bottom so that you see the newest entries.

**Comment by rejetto** [comment]
so you are talking about this page
<img width="553" alt="image" src="https://user-images.githubusercontent.com/1367199/223178435-47700d37-37fb-4036-bce3-1f095cfe591e.png">

sure, i guess I will add something for upload

**Comment by rejetto** [comment]
for the log, you just need to sort by timestamp (down), and you will always see the latest entries, like this
<img width="271" alt="image" src="https://user-images.githubusercontent.com/1367199/223463212-70a2f6c5-ed19-43bc-8e01-4ce9ba6db0d9.png">

**Comment by rejetto** [comment]
for the upload, 0.38.2 has this
<img width="466" alt="image" src="https://user-images.githubusercontent.com/1367199/223463510-6cef89d7-67b4-4296-af70-36ee97cafbd6.png">
